### ISSUE TITLE
[Project] How is startup funding distributed (geography/industry) in the US?

### ISSUE THREAD

**Issue by jlstro** [jsoma/data-studio-projects#120]
Please complete all of the following sections, or the ghost of Joseph Pulitzer will spookily dance around your issue! A completed version of this template can be found at https://github.com/jsoma/data-studio-projects/issues/1

# Pitch

## Summary
I am interested in the distribution of startup funding by industry and/or geographical area, so I downloaded startup funding data from Techcrunch and will now take a look at it.

My data came from [Techcrunch](http://techcrunch.com)

## Details

**Possible headline(s):** Silicon Valley sucks dry the start-up economy throughout the country

**Data set(s):**  http://samplecsvs.s3.amazonaws.com/TechCrunchcontinentalUSA.csv

**Code repository:** https://github.com/jlstro/Data-Studio/tree/master/code/01-startups
	
**Possible problems/fears/questions:** The data is really old and I need to get hold of a more recent data set.

## Work so far
I created a few charts and had a first look at the data.
![bildschirmfoto von 2018-07-11 12-26-33](https://user-images.githubusercontent.com/3785873/42586322-d89008c4-8505-11e8-9df2-6358ac875e5c.png)

## Checklist

This checklist must be completed before you submit your draft.

- [ ] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [ ] My issue issue has `[Project]` in the title
- [ ] My issue links to my data set(s)
- [ ] My issue links to my code repository
- [ ] My issue explains what I'd like to explore in the data set
- [ ] My issue includes images - either inspiration or what I've done so far
- [ ] I have received two comments of peer feedback on my pitch issue

**Comment by Yuanqi-Hong** [comment]
I like the idea of grouping some states into that other category so that readers are more focused on the main players.
A scatter plot with the x being industries and y being the funding would be fun!

**Comment by jlstro** [comment]
Here's some news: I spent some time scraping a website for fresh data and I have a much better data set now. This will improve my ability to analyse the data a lot. 
Attached are a few graphical ideas that I want to improve further in the next days. 

![bars-countries](https://user-images.githubusercontent.com/3785873/42720533-9ebb1ea2-86f6-11e8-8b93-9b3981146489.png)
The scale is wrong here. Need to change the colors as well.


![bars-companies](https://user-images.githubusercontent.com/3785873/42720535-a0d10e0e-86f6-11e8-8744-487a0933b792.png)
Scale also wrong, needs to be updated. Also


![pie](https://user-images.githubusercontent.com/3785873/42720536-a29e0138-86f6-11e8-8b06-c9ff7034b0af.png)
Here comes Pacman. I like the idea of showing how brutally overfinanced the US are compared to the other continents if it comes to venture capital.
![square](https://user-images.githubusercontent.com/3785873/42720537-a55755e6-86f6-11e8-81c6-d1d317e1a734.png)
Colors are stupid, but the relations are interesting. California, then nothing for a long time...
![sum-time](https://user-images.githubusercontent.com/3785873/42720539-a7a84b52-86f6-11e8-898c-b992a9877879.png)
I'll have to remove 2018 from this one.

To come:

- a deeper look at the individual industries, over time
- buzzword analysis: when was the blockchain hot, when IoT?
- nicer looks!

**Comment by Yuanqi-Hong** [comment]
For the pie chart, maybe make it bigger, so that the labels don't get on top of each other, or combine the gray guys to be 'the rest of the world'

**Comment by dz2383** [comment]
You can change 'le10' into real numbers.  We just learned how to change them today! 
And, have a legend saying the green is 'U.S' and yellow is 'non-U.S'

**Comment by jlstro** [comment]
Here's what I did on Sunday... Some new Charts!
![1](https://user-images.githubusercontent.com/3785873/42770353-9fe06a3c-88f2-11e8-96b5-ad51b396191a.png)
![2](https://user-images.githubusercontent.com/3785873/42770354-a00cdb6c-88f2-11e8-973f-20294e45629f.png)
![3](https://user-images.githubusercontent.com/3785873/42770355-a01efd9c-88f2-11e8-90e1-e809c70470ce.png)
![5](https://user-images.githubusercontent.com/3785873/42770356-a02f9170-88f2-11e8-8cb6-2e3dc2b41884.png)
![6](https://user-images.githubusercontent.com/3785873/42770357-a04b469a-88f2-11e8-876b-32d0f180b335.png)
![7](https://user-images.githubusercontent.com/3785873/42770358-a05cc4a6-88f2-11e8-8bfd-eb26444947aa.png)
![8](https://user-images.githubusercontent.com/3785873/42770359-a07b7a54-88f2-11e8-8b37-e53e945b6f54.png)

**Comment by jlstro** [comment]
There is a [live version on the server](http://167.99.158.149/01-funding/index.html) now. Lot of _lorem ipsum_ remains at that point, but I think it looks ok.

I will update my repository on github with the latest code after some minor tweaks.

**Comment by playfairbot** [comment]
Hello! I'm a little robot, just having a little look around.

You need some feedback, let me summon @SiruiZhu, @christina10211, @pasiegrist for you

It looks like we need to **fix up your [your update](https://github.com/jsoma/data-studio-projects/issues/120#issuecomment-405769691)** a little bit! Edit it by clicking the pencil in the top right-hand corner. It requires:

* New, updated content for your project
* The checklist (be sure you're using the template!)
* Exported, cleaned-up visuals of your project. Screenshots are okay if it's interactive. Links are no good, though - upload by editing your post and dragging the image in.

Maybe you just didn't use [the template](https://github.com/jsoma/data-studio-projects/blob/master/UPDATE_TEMPLATE.md)? If not, edit your comment, cut and paste the template in, and then fill it out.

**Comment by christina10211** [comment]
I can definitely see how much effort you put in through all the updates! Great job! Here're some suggestions:

- It might be better if you change all the scientific notation to something more easily readable like millions maybe?
- I love your blue color scheme, but I'm not sure if grey is the best background color when putting together with blue tho, but it might just be me. 
- The scatter plot (the one with verticle dots) is amazing, but I am a little bit confused about what each color represent in that graph, or are they just some random colors?
 - The pie chart is also great! The only thing that I would want to improve is the color. Maybe color the pie by continents or total amount of funding can make more sense here? Cuz I am a little confused about your current color representation here. 

Overall it's such a great project! I have so much fun reading through your work!

**Comment by jlstro** [comment]
# Update
	
## Your project content: images/words/etc
I created a Github Projects page, added a _little_ bit of reporting/writing. The whole thing is live now [here](https://jlstro.github.io/Data-Studio/code/01-startups/page/index.html). Some Screenshots attached.
![grafik](https://user-images.githubusercontent.com/3785873/43036575-493c5798-8cd2-11e8-8d93-265752f8230a.png)
![grafik](https://user-images.githubusercontent.com/3785873/43036578-57bcb6a0-8cd2-11e8-8514-c30f67664df9.png)
![grafik](https://user-images.githubusercontent.com/3785873/43036582-6889c8e2-8cd2-11e8-89ea-677d49a97035.png)  
## Any changes in direction or topic?
When I started, I thought I was going to be able to plot change by time. Unfortunately, my data did not allow this, which I only discovered later in the process. There are time data points in there, but they are not consistent enough.
## Problems/Questions
I had serious problems scraping the page in the beginning, to collect the data set initially. This is why I eventually focused on the top 400 or so companies. I scraped info on more than 10.000 companies only to find out that most of the data is of very poor quality.
## Checklist
	
- [x] I have included my visuals
- [x] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by pasiegrist** [comment]
Wow great work. I would make some design adjustments. It's mostly about coloring:
 
- I would choose a little lighter gray for the background. It's too heavy for me.
- Id change the coloring of the american and non-american company. My I is drawn to the brighter color, but your focus is on the USA.
- And sorry, I don't like Pacman. I hated it first, then read your argument, which kind of convinced me at first, but… no sorry. :)

**Comment by SiruiZhu** [comment]
I like this idea, and they way you trying to pursue it! I can see the changes you make through is great. Here I come up with some thinkings might be helpful or not:

- For your last chart, like annotation we did that day with Bui, it might be better if you move the second one" Chinese...Alibaba..." to the top, so there is a hierarchy to read the annotation left to right. Now，it's a little hard to follow with that one. Like this: 
<img width="664" alt="1" src="https://user-images.githubusercontent.com/39416241/43047842-aee3d34e-8dab-11e8-8e85-128a366cc0c1.png">

- For the last chart and the rest, the fontsize is little bit small to me, because you have many texts in this image, so it's nice to have a fontsize hierarchy too, make them different by giving them different fontsize, probably will be better for reading?

Overall, I really enjoy reading your project!

**Comment by sarahslo** [comment]
part of the problem i'm having with the pie charts, when you have a big number that overwhelms the rest you lose the ability to see the nuance. i'd lose those charts altogether. 
also the big chart showing companies by valuation, you could simply put all the companies together by country and then we'd get a better sense of china vs the us. i'm not sure really tho, why i care to compare these companies as they do such different things. who is the audience here? 

i think the last chart has potential. it actually takes the data and tells me something that couldn't be told in a sentence. i'd like to know more about each category. you could go into this data in more detail as separate charts. 
also your annotations need to inform me. knowing that xiaomi is hyped or fintech companies spring up like mushrooms..doesn't tell me anything if i don't know anything about the company or the category to begin with.

i'd like to see you respect your audience a little more and help us understand what is important here and why.

**Comment by jlstro** [comment]
Sarah, thanks for your feedback. I actually liked the idea that the pie charts "overwhelm" a bit - the message here is that the US are huge compared to the rest. But you are right, the nuances get lost this way. I kinda tried to put all together by company in the pie charts, maybe a bar is the better option here.

Just to clarify: Are you referring to the screenshots only or the actual page I made, with what you wrote? I had a feeling that the text around the charts on my page might support with the context and also help with some of the problems you mention above...

**Comment by jlstro** [comment]
# Final
## Project visuals/text
"Unicorn, noun - a start-up that is valued at one Billion dollars or more" (Merriam Webster). When serial investor Aileen Lee coined the term Unicorn for a billion-dollar-valuation more than five years ago, these companies where rare - not as rare as the term suggests, though. Over the past years, startups have evolved into well-known brands and hidden champions have developed serios valuations. This analysis tries to make use of data to show which industries are the biggest and which region is the most fruitful.
![grafik](https://user-images.githubusercontent.com/3785873/43107848-c88a19f0-8eac-11e8-8eda-a22a303333cf.png)
![grafik](https://user-images.githubusercontent.com/3785873/43107861-dda92394-8eac-11e8-8c14-f298f44defc0.png)
![grafik](https://user-images.githubusercontent.com/3785873/43107881-f66a9eda-8eac-11e8-9672-4b256a8f3224.png)
![grafik](https://user-images.githubusercontent.com/3785873/43107891-07f451fa-8ead-11e8-9b7d-1a366884dadd.png)
![grafik](https://user-images.githubusercontent.com/3785873/43107900-1362f546-8ead-11e8-938d-5b8b8a239527.png)

## Details
**Headline:** 
🦄 & runner-ups
**Published website version:**
[Here](https://jlstro.github.io/Data-Studio/code/01-startups/page/index.html)
**Code repository:**
[Here](https://github.com/jlstro/Data-Studio/tree/master/code/01-startups)
**Final data set(s):** 
See repository
## What did you find to be the most difficult part of this project?
Working with illustrator (and finding the data, or rather deciding on what data I wanted to work with)
## Are you satisfied with what you produced? Is there anything you would like to change or improve?
It is not what I initially wanted to do, especially as the end result does not say anything about a development over time. But it looks ok, I guess..
## Checklist
- [x] I have included my visuals
- [x] I have posted my project to my project website
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository
